lightning/src/ln/chan_utils.rs

```diff
@@ -540,7 +546,13 @@ pub(crate) fn get_htlc_redeemscript_with_explicit_keys(htlc: &HTLCOutputInCommit
 		              .push_opcode(opcodes::all::OP_CLTV)
 		              .push_opcode(opcodes::all::OP_DROP)
 		              .push_opcode(opcodes::all::OP_CHECKSIG)
-		              .push_opcode(opcodes::all::OP_ENDIF)
+		              .push_opcode(opcodes::all::OP_ENDIF);
+			if opt_anchors {		 
+					bldr = bldr.push_opcode(opcodes::all::OP_PUSHNUM_1)
```

Comment:
this is indented two tabs instead of one, and also the next lines

Reply:
fixed